src/backends/kimchi/asm.rs

```diff
@@ -90,10 +90,12 @@ where
         self.map[value]
     }
 
+    #[must_use]
     pub fn len(&self) -> usize {
         self.ordered.len()
     }
 
+    #[must_use]
```

Comment:
wondering what was the lint for these

Reply:
Turns out this might've been a false positive. I reverted the changes to this file.